sim/src/analytics.rs

```diff
@@ -52,6 +53,9 @@ pub struct Analytics {
     /// If it is over a certain threshold (<95% of max speed)
     /// TripID, [(LaneID, Percent of maximum speed as an integer (0-100)]
     pub lane_speed_percentage: BTreeMap<TripID, BTreeMap<LaneID, u8>>,
+    /// Record every instance of somebody on foot or a bike crossing an intersection with more than
+    /// 4 connecting roads.
+    pub large_intersection_crossings: BTreeMap<TripID, Vec<(Time, IntersectionID)>>,
```

Comment:
Negligible increase for prebaked file size -- 61MB to 62MB for the largest, lakeslice